### ISSUE TITLE
setAttrs doesn't work and not update AndroidManifest

### ISSUE THREAD

**Issue by KevinToala** [ionic-team/capacitor-configure#10]
When use setAttrs for modify the AndroidManifest values the final result remove all data in my AndroidManifest
project.android?.getAndroidManifest() .setAttrs( "/manifest/application/meta-data[@*='com.google.android.geo.API_KEY']",
      {
        'android:value': '---API-KEY---',
      }
    );

![image](https://user-images.githubusercontent.com/16714689/140444580-84bf3304-4a36-4d01-9f79-ec9879bec4fe.png)

I was reviewing the code and it seems that the problem is in the following toString () method. It is empty and does nothing

https://github.com/ionic-team/capacitor-configure/blob/d6d939ae2092b416905e3ae7f4e5d6db4973ce73/packages/project/src/android/manifest.ts#L24

**Comment by mlynch** [comment]
Taking a look. You're on 1.0.9?

**Comment by mlynch** [comment]
Found the issue, fixing. Sorry about this

**Comment by mlynch** [comment]
Fixed in 1.0.10 for `@capacitor/configure` and `@capacitor/project`. Thanks again, sorry for the trouble, and will expand the test suite to catch issues like this so they don't happen again.

**Comment by KevinToala** [comment]
Hi @mlynch.

thank you for you support.

I just upgraded to the new version and I get the same behavior. Check the code and I see that it follows the same line of the problem. Maybe it was not correctly uploaded to npm?

![image](https://user-images.githubusercontent.com/16714689/140561774-9292e916-abec-4852-97e0-d1b5e09e5708.png)

**Comment by mlynch** [comment]
Gah, sorry about that, didn't properly build the assets for publish. Fix is live in 1.0.11

**Comment by KevinToala** [comment]
Awesome. Now is working. Thanks @mlynch :) 

I have a little question. I don't know if is in the correct thread, sorry.

My project is with angular / ionic and capacitor. Where should I have these @capacitor/project settings?
Currently I have them in the capacitor.config.ts file and it runs automatically whenever I run a command npx cap copy, npx cap sync, etc ... is this fine? or how should it work?

**Comment by mlynch** [comment]
I think it's fine if it works for you. I wanted to just see how people wanted to use this before developing a stronger opinion on how people _should_ use it.

I don't love the idea of applying more complex configuration to your app that is already in there, like manifest changes. Those feel like they should be run more periodically and only when something explicitly changes. However, if done right, those shouldn't cause any issue for your app.

So, basically, I'm just waiting and gathering feedback before recommending any specific way to use the tool

**Comment by KevinToala** [comment]
I understand, thanks for the comment.
Finally create a separate file and with the following script in package.json I execute the changes only when required and not always.
`"cap:sync:environment": "tsc --resolveJsonModule capacitor.project.ts && node capacitor.project.js"`

I will be following up on this project. I found it very interesting
